gameplay/src/Vector2.h

```diff
@@ -413,6 +421,15 @@ class Vector2
     inline bool operator<(const Vector2& v) const;
 
     /**
+    * Determines if this vector is greater than the given vector.
+    *
+    * @param v The vector to compare against.
+    *
+    * @return True if this vector is greater than the given vector, false otherwise.
+    */
+    inline bool operator>(const Vector2& v) const;
```

Comment:
Sure. I'm using it.
As comparison goes, glm even has bitwise operator for vectors...